patroni/config.py

```diff
@@ -242,7 +242,8 @@ def _set_section_values(section, params):
                 if value:
                     ret[section][param] = value
 
-        _set_section_values('restapi', ['listen', 'connect_address', 'certfile', 'keyfile', 'cafile', 'verify_client'])
+        _set_section_values('restapi', ['listen', 'connect_address', 'certfile', 'keyfile', 'cafile', 'verify_client',
+                                        'http_extra_headers', 'https_extra_headers'])
```

Comment:
Please also update [ENVIRONMENT.rst](https://github.com/zalando/patroni/blob/master/docs/ENVIRONMENT.rst) similar to https://github.com/zalando/patroni/pull/1645/files#diff-40772a77bd7d4ea1f3cb4cbef17a813dR274-R275

Reply:
Updated